README.md

```diff
@@ -161,6 +161,34 @@ will be added as they are required.
 
 ---
 
+`should_not_redirect`
```

Comment:
In this case, I have the irule changing the HTTP::uri before passing it to a pool.

So, in this case, the I am using should_select_pool

But I also want to be able to validate that the URI itself is being set correctly.

The irule is making changes much like a PT rewrite in Apache.  The browser is none the wiser that the request was passed to the app server differently than what was requested.  (SEO team’s requirements)

Jon Abend
C (312) 446-3914

From: Jesse Mauntel [mailto:notifications@github.com]
Sent: Monday, March 09, 2015 10:13 AM
To: jmauntel/irule-tester
Cc: Abend, Jon
Subject: Re: [irule-tester] Added two new functions and updated two existing to extend framework (#1)


In README.md<https://github.com/jmauntel/irule-tester/pull/1#discussion_r26043408>:

> @@ -161,6 +161,34 @@ will be added as they are required.

>

>  ---

>

> +`should_not_redirect`

I'm not sure in what case you would need to do this. Since each request can only be responded to by the F5 once, wouldn't using one of the other tests that call out what you want it to do be better? For instance "should_select_pool"? Am I missing a use case here?

—
Reply to this email directly or view it on GitHub<https://github.com/jmauntel/irule-tester/pull/1/files#r26043408>.